### ISSUE TITLE
Unexpected indentation warning for setting variables of an object returned by a function.

### ISSUE THREAD

**Issue by liutikas** [checkstyle/checkstyle#255]
I am using the following configuration:

``` xml
<module name="Indentation">
      <property name="basicOffset" value="4"/>
</module>
```

Java code:

``` java
class HelloWorldApp {
    public void setX(int newX) {
        getItem().x = newX;
    }
}
```

Checkstyle gives me the following warning:
HelloWorldApp.java:3: warning: method call child at indentation level 8 not at correct indentation, 12

It seems to me that is a bug since there is no reason to indent entire getItem().x = newX line by 4 more space.

Thoughts?
